csharp-basics/exercises/Arithmetic/Exercise 8/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿decimal salary1;
+decimal salary2;
+decimal salary3;
+
+int hours1;
+int hours2;
+int hours3;
```

Comment:
saliecdefinīciju ar vertības piešķiršanu kopā. :)

Reply:
Šis bija izdarīts tā pat kā lekcijā, bet okey